src/main/java/frc/utils/GBSubsystem.java

```diff
@@ -6,6 +6,12 @@
 
 public abstract class GBSubsystem extends SubsystemBase {
 
+    private final String loggingPath;
+
+    public GBSubsystem(String loggingPath) {
+        this.loggingPath = "Subsystems/" + loggingPath;
+    }
+
     @Override
     public final void periodic() {
         Logger.recordOutput(getLogPath() + "Current Command", getCurrentCommandName());
```

Comment:
Why are you doing getLogPath if loggingPath is already a variable in this class?

Reply:
We can do public final but its ugly

Reply:
I don't want you to switch to public final, but you can just use loggingPath

Reply:
I meant only in this line change it